esignet-service/src/main/resources/application-default.properties

```diff
@@ -91,6 +91,8 @@ mosip.esignet.additional-config.schema.url=classpath:additional_config_request_s
 ## Transaction halted with prepare-signup-redirect wait time to resume back with complete-signup-redirect API
 mosip.esignet.signup.halt.expire-seconds=1800
 
+server.servlet.session.timeout=30m
```

Comment:
We donot use HttpSessionCsrfTokenRepository, we use our custom CookieCsrfTokenRepository, pls check